tests/cypress/e2e/textFieldInitializerTest.cy.ts

```diff
@@ -0,0 +1,455 @@
+        pageComposer.switchLanguage('English');
+        checkValuesDisplayedInPageComposer(pageComposer, valuesToCheck, langEN);
+    });
+
+    it('Verify the deletion of previous text initializer', () => {
+        const fieldsToEdit = {
+            'jnt:textFieldInitializer_defaultDate': {
+                type: 'date',
+                values: {
+                    en: '12/07/1998 19:40',
+                    fr: '12/07/1998 19:40',
+                    de: '12/07/1998 19:40'
+                }
+            },
+            'jnt:textFieldInitializer_defaultI18nDate': {
+                type: 'date',
+                values: {
+                    en: '12/07/2002 19:40',
+                    fr: '12/07/2000 19:40',
+                    de: '12/07/2004 19:40'
+                }
+            },
+            'jnt:textFieldInitializer_defaultDateAutocreated': {
+                type: 'date',
+                values: {
+                    en: '07/03/2009 19:40',
+                    fr: '07/03/2009 19:40',
+                    de: '07/03/2009 19:40'
+                }
+            },
+            'jnt:textFieldInitializer_defaultI18nDateAutocreated': {
+                type: 'date',
+                values: {
+                    en: '07/05/2008 19:40',
+                    fr: '07/04/2008 19:40',
+                    de: '07/06/2008 19:40'
+                }
+            }
+        };
+
+        const valuesToCheck = {
+            fr: ['1998-07-12T19:40:00.000', '2000-07-12T19:40:00.000', '2009-03-07T19:40:00.000', '2008-04-07T19:40:00.000'],
+            de: ['1998-07-12T19:40:00.000', '2004-07-12T19:40:00.000', '2009-03-07T19:40:00.000', '2008-06-07T19:40:00.000'],
+            en: ['1998-07-12T19:40:00.000', '2002-07-12T19:40:00.000', '2009-03-07T19:40:00.000', '2008-05-07T19:40:00.000']
+        };
+
+        cy.login();
+        const pageComposer = PageComposer.visit(siteKey, langEN, 'home.html');
+        pageComposer.openContextualMenuOnContent('div[path="/sites/extFieldInitializerTest/home/area-main/textfieldinitializer"] div.gwt-HTML.x-component').delete();
+        pageComposer.openContextualMenuOnContent('div[path="/sites/extFieldInitializerTest/home/area-main/textfieldinitializer"] div.gwt-HTML.x-component').deletePermanently();
+        const contentEditor = pageComposer
+            .openCreateContent()
+            .getContentTypeSelector()
+            .searchForContentType('textFieldInitializer')
+            .selectContentType('textFieldInitializer')
+            .create();
+        editFieldValues(contentEditor, fieldsToEdit, langEN);
+        contentEditor.getLanguageSwitcher().selectLang('Français');
+        editFieldValues(contentEditor, fieldsToEdit, langFR);
+        contentEditor.getLanguageSwitcher().selectLang('Deutsch');
+        editFieldValues(contentEditor, fieldsToEdit, langDE);
+        contentEditor.getLanguageSwitcher().selectLang('English');
```

Comment:
you could do these checks in a separate function within a loop for each language

Reply:
Done